juju/juju
# Use offer-uuid to look up offer permissions and send messages between models

## Description of change

Use the UUID of the offer instead of the name to:
- communicate between models
- add as a first party caveat to macaroons
- look up access permission

## QA steps

Run up a cmr scenario
Ensure relation works
Add user, grant permission
Ensure added user can relate to the offer
